openmc/surface.py

```diff
@@ -335,29 +412,45 @@ def z0(self, z0):
         self._coeffs['z0'] = z0
 
 
-
 class Cylinder(Surface):
+    """A cylinder whose length is parallel to the x-, y-, or z-axis.
```

Comment:
You know I just realized that the ``Cylinder`` class should be abstract since a generic version does not exist in OpenMC. This is separate from this PR's focus on docstrings / PEP8 compliance, but it should be easy to add the following to this class to prevent uses from instantiating it:

```python
  # This is an abstract class which cannot be instantiated
  metaclass__ = abc.ABCMeta
```